tests/pages/api/register.test.ts

```diff
@@ -267,4 +284,46 @@ describe('register', () => {
             res,
         );
     });
+
+    it('should update user attributes with contactable if given', async () => {
+        authSignInSpy.mockImplementation(() => Promise.resolve(mockAuthResponse));
+        authCompletePasswordSpy.mockImplementation(() => Promise.resolve());
+        authSignOutSpy.mockImplementation(() => Promise.resolve());
+        authUpdateAttributesSpy.mockImplementation(() => Promise.resolve());
```

Comment:
We should be good to move these mockImplementations into the beforeEach as all of the tests use the same implementations.

Not something raised in this PR, but the mockImplementation on line 166 could be removed